lib/team.rb

```diff
@@ -0,0 +1,28 @@
+require 'pry'
+class Team
+  attr_reader :country,
+              :players
+
+  attr_accessor :eliminated
```

Comment:
You should not need an attr_accessor here - you have a method below with the correct name!